vms/platformvm/state/state.go

```diff
@@ -1944,3 +2010,50 @@ func (s *state) writeMetadata() error {
 	}
 	return nil
 }
+
+// TODO: Remove after next hard-fork when everybody has run this.
+func (s *state) populateBlockHeightIndex() error {
+	blk, _, err := s.GetStatelessBlock(s.lastAccepted)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve last accepted block from local disk: %w", err)
+	}
+
+	heightKey := database.PackUInt64(blk.Height())
+	if _, err := database.GetID(s.blockIDDB, heightKey); err == nil {
```

Comment:
```suggestion
	lastAcceptedHeight := database.PackUInt64(blk.Height())
	if _, err := database.GetID(s.blockIDDB, lastAcceptedHeight); err == nil {
```

to be more readable?